src/neo/Network/P2P/Payloads/TransactionAttributeType.cs

```diff
@@ -17,6 +17,24 @@ public enum TransactionAttributeType : byte
         /// Indicates that the transaction is an oracle response.
         /// </summary>
         [ReflectionCache(typeof(OracleResponse))]
-        OracleResponse = 0x11
+        OracleResponse = 0x11,
+
+        /// <summary>
+        /// Indicates that the transaction is not valid before <see cref="NotValidBefore.Height"/>.
+        /// </summary>
+        [ReflectionCache(typeof(NotValidBefore))]
+        NotValidBefore = 0xe0,
```

Comment:
Just noting that all constants currently used by notary subsystem were chosen mostly to prevent intersections with other protocol parts (see #1904 also), so they may seem a little strange, but they can be changed if we're to settle on this proposal.